--- pyftpdlib-1.5.5/pyftpdlib/test/__init__.py.orig	2018-05-15 14:14:07.000000000 +0200
+++ pyftpdlib-1.5.5/pyftpdlib/test/__init__.py	2019-10-05 17:47:17.336707773 +0200
@@ -37,7 +37,8 @@
     unittest.TestCase.assertRaisesRegex = unittest.TestCase.assertRaisesRegexp
 
 if os.name == 'posix':
-    import sendfile
+    if not hasattr(os, 'sendfile'):
+        import sendfile
 else:
     sendfile = None
 
--- pyftpdlib-1.5.5/pyftpdlib/test/test_functional.py.orig	2019-03-27 17:31:43.000000000 +0100
+++ pyftpdlib-1.5.5/pyftpdlib/test/test_functional.py	2019-10-05 17:55:46.157284589 +0200
@@ -67,7 +67,8 @@
 import ssl
 
 if POSIX:
-    import sendfile
+    if not hasattr(os, 'sendfile'):
+        import sendfile
 else:
     sendfile = None
 
